app/mailers/bulk_upload_mailer.rb

```diff
@@ -0,0 +1,79 @@
+class BulkUploadMailer < NotifyMailer
+  BULK_UPLOAD_COMPLETE_TEMPLATE_ID = "83279578-c890-4168-838b-33c9cf0dc9f0".freeze
+  BULK_UPLOAD_FAILED_CSV_ERRORS_TEMPLATE_ID = "e27abcd4-5295-48c2-b127-e9ee4b781b75".freeze
+  BULK_UPLOAD_FAILED_FILE_SETUP_ERROR_TEMPLATE_ID = "24c9f4c7-96ad-470a-ba31-eb51b7cbafd9".freeze
+  BULK_UPLOAD_FAILED_SERVICE_ERROR_TEMPLATE_ID = "c3f6288c-7a74-4e77-99ee-6c4a0f6e125a".freeze
+  BULK_UPLOAD_WITH_ERRORS_TEMPLATE_ID = "eb539005-6234-404e-812d-167728cf4274".freeze
+
+  def send_bulk_upload_complete_mail(user, bulk_upload)
+    send_email(
+      user.email,
+      BULK_UPLOAD_COMPLETE_TEMPLATE_ID,
+      {
+        title: "[#{bulk_upload} title]",
+        filename: "[#{bulk_upload} filename]",
+        upload_timestamp: "[#{bulk_upload} upload_timestamp]",
```

Comment:
if the things being interpolated/inserted here is a `Time` object, how would we like it formatted? And is that going to be part of another ticket or is the time formatting linked enough to the templating in this ticket to be worth doing now?
Also is there a standard gov or CORE set of date presentation formats?